src/components/StudentGrid.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import ButtonBase from '@material-ui/core/ButtonBase';
+import logopic from '../img/logo.png';
+import LoginButton from './LoginButton';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  paper: {
+    padding: theme.spacing(2),
+    margin: 'auto',
+  },
+  image: {
+    width: 750,
+    height: 250,
+  },
+  img: {
+    margin: 'auto',
+    display: 'block',
+    maxWidth: '100%',
+    maxHeight: '100%',
+  },
+}));
+
+export default function StudentGrid() {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
```

Comment:
Use a [`<Container>`](https://material-ui.com/components/container/) here

Reply:
Ok it is now changed to `<Container>` inline I also changed the textAlign to be 'left' so that it matches with the Figma design that we currently have.